crates/bindings-csharp/Codegen/Module.cs

```diff
@@ -31,18 +33,29 @@ bool isEquatable
     )
         : base(name, type, typeInfo)
     {
-        Attrs = attrs;
+        (string?, ColumnAttrs)[] x = [(default(string), attrs)];
```

Comment:
What's happening here?

Reply:
[].ToImmutableArray() doesn't compile without this type hint.

Reply:
I meant more generally. Why do we create this `x` with empty table and provided attrs in the first place?

Reply:
Ah this is for fake scheduling fields.

Hm can you just extract `(string? table, ColumnAttrs mask)` into `record struct` after all? It will be basically the same, but a bit more readable in both debug view (which doesn't preserve tuple field names) and because you'll be able to name the type here more clearly.

Also take a look at https://learn.microsoft.com/en-us/dotnet/api/system.collections.immutable.immutablearray.create?view=net-8.0#system-collections-immutable-immutablearray-create-1(-0) which will simplify this one step further.

Reply:
Ah nice, didn't know about IA.Create